test/rostest_avoidance_run.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+PX4_SRC_DIR=${DIR}/..
+
+source /opt/ros/${ROS_DISTRO}/setup.bash
```

Comment:
This is good for Kinetic (Gazebo 7) and Melodic (Gazebo 9)? Using Kinetic for now, so maybe change to:
`source /opt/ros/${ROS_DISTRO:-kinetic}/setup.bash`, matching https://github.com/PX4/Firmware/blob/2ae2331c15e10008f01c55d90f549cae91716acb/test/rostest_px4_run.sh#L6